stdlib/array.gr

```diff
@@ -871,3 +871,38 @@ export let slice = (startIndex, endIndex, array) => {
     init(newLength, n => array[startIndex + n])
   }
 }
+
+/**
+ * Sorts an array in-place using the quicksort algorithm.
+ * 
+ * @param comp: (a, a) -> Number - A function that takes two arguments. It must return 0 if its arguments compare as equal, a positive number if the first is greater, and a negative number if the first is smaller
```

Comment:
```suggestion
 * @param comp: The comparator function used to indicate sort order
```
Our graindoc tool actually uses the inferred type, so we don't specify it in the docs. Also, I think the description of the function should discuss the comparator functions instead of the argument.